Generate a consistent api_key for user, system, or classic auth_types every time (#806)
RedHatInsights/insights-host-inventory: README.md

To force an ssl connection to the db set INVENTORY_DB_SSL_MODE to "verify-full"
and provide the path to the certificate you'd like to use.

## Generating API Keys

> For creating and accessing hosts in the insights-inventory database, it is important to know which api key was used to create a host.  There have been occasions when a considerable amount of time was spent to determine why the hosts in inventory database were not accessible using REST API.  The reason turned out was using wrong apikey for accessing hosts.  To avoid such problems, generate the same apiKey every time using the same input json for a desired authentication type:

Reviewer: I think this could be made more concise. Also some suggestions for the sake of consistance (e.g. apiKey vs API key, and json to JSON.
```suggestion
When accessing hosts the inventory API, it is important to know which API key was used to create the host. If you are not seeing the hosts you have created in responses from the API an API key mismatch may be the cause. To avoid such problems, generate the same API key every time using the same input JSON for a desired authentication type:
```

```
python utils/create_api_key.py cert
```

> It outputs a base64-encode apiKey alongwith the json used for generating the key

Reviewer: just some little typos and changes to maintain consistency. 
```suggestion
This outputs a base64 encoded API key along with the JSON used for generating the key
```
